apps/asap-server/src/app.ts

```diff
@@ -75,6 +76,8 @@ export const appFactory = (libs: Libs = {}): Express => {
   // Handlers
   const authHandler = libs.authHandler || authHandlerFactory(decodeToken);
   const tracingHandler = tracingHandlerFactory(libs.tracer);
+  const sentryTransactionIdHandler =
```

Comment:
you're not testing it anyway so i would just do `app.use(<< handler here>>)` without looking it up in libs